Adds PT-BR Localization

brwolfgang opened the pull request:
Initial pull request adding PT-BR localization containing all mandatory files as specified on the [Translation Guide](https://github.com/letsencrypt/website/blob/master/TRANSLATION.md)

schoen commented:
Hi @brwolfgang,

I'm a non-native speaker of Brazilian Portuguese and I work on the Let's Encrypt project.

Thanks for your translation contributions here!

I've read your translation and I have a small number of questions.

(1) You referred to Let's Encrypt as "o Let's Encrypt". I've normally used "a Let's Encrypt" in talking about the project, thinking of it as "a [autoridade certificadora] Let's Encrypt". Do you think the masculine article is widely-used or well-established among native Portuguese speakers or in existing Portuguese documentation about the project? Do you think the feminine article may be reasonable?

(2) I created a Portuguese-specific category on the Community Forum (inspired by an existing French category); you can find it at https://community.letsencrypt.org/c/help/ajuda-em-portugues and I think this might be a better link from `Obter Ajuda` than the top-level https://community.letsencrypt.org/ (where the great majority of the discussion takes place in English). So far almost nobody has ever found or used the "Ajuda (em português)" forum category, but that might change if we had a direct link to it from the Let's Encrypt site!

(3) In `_index.md`, you stopped translating section titles at "Como o Let's Encrypt Funciona"; do you think it would be useful to have translations for the rest of the section titles?

(4) In `technicaloverview.md`, perhaps `protocolo ACME` should be `o protocolo ACME`, and `certrificado` should be `certificado`.

(5) In the same file, the paragraph beginning `Se a assinatura no pacote de dados é válida` is missing closing punctuation at the end of the paragraph.

brwolfgang commented:
I agree with most points, gonna update it and let you know when ready.

About (3) I only translated the title of pages with corresponding PT-BR localizations, from personal experience on other projects it would be confusing to users reading the translated title here and then land on the English version. As the other pages get translated I will update this index.

brwolfgang commented:
@schoen I believe it's ready for your review.

schoen commented on content/pt-br/about.md:
```diff
@@ -15,11 +15,11 @@ Nós fornecemos gratuitamente os certificados digitais necessários para habilit
 
 Os princípios chave do Let's Encrypt são: 
```
"da Let's Encrypt" :-)

brwolfgang replied:
Weird how my text search engine didn't spot this, gonna fix it shortly.

schoen commented:
Thanks, I might be fighting a losing battle against native speakers' intuitions here...

(You missed the very first reference to "do Let's Encrypt" in the sentence "Os princípios chave do Let's Encrypt são".)

Your explanation of point 3 makes sense to me now, and thanks for all the translation work!

schoen commented:
It looks like usage online is very mixed; with forms of the verb `usar`, "o Let's Encrypt" dramatically predominates (like "usar o Let's Encrypt"), but with the noun `certificado`, "a Let's Encrypt" is also common (like "certificado da Let's Encrypt"). I'm not sure how to account for this different.

Wikipedia uses primarily "a Let's Encrypt" except in one paragraph ("O projeto Let’s Encrypt começou em 2012 por dois empregados da Mozilla, Josh Aas e Eric Rescorla, junto com Peter Eckersle da Eletronic Frontier Foundation e J. Alex Halderman da Universidade de Michigan. A Internet Security Research Group, empresa por trás do Let’s Encrypt, foi incorporada em Maio de 2013.") which is regarding it as a "projeto" as opposed to an "autoridade certificadora"—and also misspelling @pde's name :disappointed:.

brwolfgang commented:
You-re right, at first I was mentioning Let's Encrypt as male as in "O serviço Let's Encrypt", but end-up mixing things up. I modified my text query and ended up finding other issues, that I have now fixed.

schoen commented:
I still see two instances:

https://github.com/letsencrypt/website/pull/496/files#diff-4408d5ef4445690b8ecbe236cac70623R53
and
https://github.com/letsencrypt/website/pull/496/files#diff-0add5aab62f13e7a45b3342c02f5cdb6R11

Also, you should exclude your `.idea` directory from the pull request—it looks like that's a local directory on your system for a text editor or version control tool, rather than something that needs to be shared with the public.

brwolfgang commented:
Oh, the joys of adding unintended files to Git! This and the remaining strings are fixed.

brwolfgang commented:
Hi there! I don't mind the mention at all, my handler is @wmbr7

jaykaypea commented:
This is so awesome, thank you so much @brwolfgang!
